cmd/resim/commands/build.go

```diff
@@ -40,11 +40,16 @@ const (
 
 func init() {
 	createBuildCmd.Flags().String(buildDescriptionKey, "", "The description of the build, often a commit message")
+	createBuildCmd.MarkFlagRequired(buildDescriptionKey)
 	createBuildCmd.Flags().String(buildImageURIKey, "", "The URI of the docker image")
+	createBuildCmd.MarkFlagRequired(buildImageURIKey)
 	createBuildCmd.Flags().String(buildVersionKey, "", "The version of the build image, usually a commit ID")
+	createBuildCmd.MarkFlagRequired(buildVersionKey)
 	createBuildCmd.Flags().String(buildProjectNameKey, "", "The name of the project to create the build in")
+	createBuildCmd.MarkFlagRequired(buildProjectNameKey)
 	createBuildCmd.Flags().String(buildBranchNameKey, "", "The name of the branch to nest the build in, usually the associated git branch")
 	createBuildCmd.Flags().Bool(buildAutoCreateBranchKey, false, "Whether to automatically create branch if it doesn't exist")
+	createBuildCmd.MarkFlagsMutuallyExclusive(buildBranchNameKey, buildAutoCreateBranchKey)
```

Comment:
actually, this is not correct (right now). The user has to specify the branch name in each case: it isn't inferred, since there's no guarantee they user is using this tool within a git repo. Auto-create will just hit the create branch endpoint if it can't find a branch with the name

Reply:
Thanks! Fixed